src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -919,4 +933,59 @@ private Predicate<String> getModulePredicate() {
   public RandomNumberGenerator getRandomizer() {
     return this.populationRandom;
   }
+
+  public void reduceNumberOfHospitals() {
```

Comment:
Can you please add a java doc to explain what the method is doing.

Reply:
Added Javadoc